R/create.R

```diff
@@ -109,34 +110,26 @@ create_shifted_cases <- function(data, shift,
   ]
 
   ## Forecast trend on reported cases using the last week of data
-  final_period <- data.table::data.table(
-    confirm =
-      shifted_reported_cases[!is.na(confirm)][
-        max(1, .N - smoothing_window):.N
-      ]$confirm
-  )[,
+  final_period <- shifted_reported_cases[!is.na(confirm)][
```

Comment:
In the spirit of recent changes there is quite a good arg this should move inside the model

Reply:
as it has always been a big hack

Reply:
yes good point